src/main.cpp

```diff
@@ -204,9 +204,11 @@ int main(int argc, const char* argv[])
 
         LOG.info("Running {}", full_version_string());
 
-        const auto curdir     = khepri::application::get_current_directory();
-        auto       data_paths = args->modpaths;
-        data_paths.push_back(curdir);
+        auto data_paths = args->modpaths;
+        data_paths.push_back(openeaw::io::PathManager::get_install_path(openeaw::io::steam) /
+                             "GameData");
```

Comment:
This should call the 'auto' variant, shouldn't it? Also, it should handle the case when EaW can't be found.